src/framework/components/anim/component-binder.js

```diff
@@ -240,10 +255,31 @@ class AnimComponentBinder extends DefaultAnimBinder {
             return new AnimTarget(function (values) {
                 setter(values);
                 propertyComponent.material.update();
-            }, animDataType, animDataComponents);
+            }, animDataType, animDataComponents, targetPath);
         }
 
-        return new AnimTarget(setter, animDataType, animDataComponents);
+        return new AnimTarget(setter, animDataType, animDataComponents, targetPath);
+    }
+
+    rebind() {
+        this.targetCache = {};
+        // #ifdef DEBUG
+        this.visitedFallbackGraphPaths = {};
+        // #endif
+        var entity = this.animComponent && this.animComponent.entity;
+        var graph = entity.model && entity.model.model && entity.model.model.graph;
```

Comment:
Again, you could potentially use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here:

```javascript
                var graph = entity.model?.model?.graph;
```

Reply:
Although I'm a bit confused about the logic here. The code checks there's an `animComponent` property without assuming it's existence. Which indicates maybe `entity` could end up as `undefined`. But `entity` is assumed to be defined on the following line.

Reply:
Yeah I didn't need to check animComponent here as it's always always available to the class. I've updated this to use conditional chaining too.